README.md

```diff
@@ -28,5 +28,7 @@ Runs `terraform plan` and `apply` remotely and comments back on the pull request
 [![Go Report Card](https://goreportcard.com/badge/github.com/runatlantis/atlantis)](https://goreportcard.com/report/github.com/runatlantis/atlantis)
 [![codecov](https://codecov.io/gh/runatlantis/atlantis/branch/master/graph/badge.svg)](https://codecov.io/gh/runatlantis/atlantis)
 [![CircleCI](https://circleci.com/gh/runatlantis/atlantis/tree/master.svg?style=shield)](https://circleci.com/gh/runatlantis/atlantis/tree/master)
+[![Docker Image Metadata](https://images.microbadger.com/badges/image/runatlantis/atlantis.svg)](https://microbadger.com/images/runatlantis/atlantis)
+[![Docker Image Version](https://images.microbadger.com/badges/version/runatlantis/atlantis.svg)](https://microbadger.com/images/runatlantis/atlantis)
```

Comment:
I don't think this badge is useful, it will always say `latest`

Reply:
I can update to these two:

```
[![](https://images.microbadger.com/badges/image/runatlantis/atlantis:v0.8.3.svg)](https://microbadger.com/images/runatlantis/atlantis:v0.8.3 "Get your own image badge on microbadger.com")
[![](https://images.microbadger.com/badges/version/runatlantis/atlantis:v0.8.3.svg)](https://microbadger.com/images/runatlantis/atlantis:v0.8.3 "Get your own version badge on microbadger.com")
```

Reply:
Let me know which one do you preferred.
[![Docker Image Metadata](https://images.microbadger.com/badges/image/runatlantis/atlantis:v0.8.3.svg)](https://microbadger.com/images/runatlantis/atlantis:v0.8.3)
[![Docker Image Version](https://images.microbadger.com/badges/version/runatlantis/atlantis:v0.8.3.svg)](https://microbadger.com/images/runatlantis/atlantis:v0.8.3)

Also, looks like there is some image size issue.

Reply:
The root cause of the versioned tag missing (not same as `latest`) is because:

The latest docker container is always the latest branch, while the versioned docker image is per release.

Reply:
* I think the layers one is cool but if it's broken, aka always showing 0B then don't think it's worth having.
* The version one will require updating on every release. I want to move towards CI/CD based releasing which will be harder if we also have to write a commit to the README so I don't think that one is worth having.